exp/utils.py

```diff
@@ -60,5 +60,7 @@ def round_age(age_in_days):
 def round_ages_from_birthdays(child_birthdays, date_created):
     return [
         round_age((date_created.date() - birthdate).days)
+        if birthdate and type(birthdate) == type(date_created.date())
```

Comment:
`type(birthdate) is datetime.date`

Reply:
yep, was foolishly trying to avoid importing datetime just for that :)